DotnetCLIVersion.txt

```diff
@@ -1 +1 @@
-2.1.301
+2.1.401-preview-009081
```

Comment:
This should be `2.1.302` (N-1) at maximum right now. I don't know if there's a plan for how to build targeting netcoreapp3.0 without it yet.

/cc @dleeapho @dseefeld @crummel

Reply:
I see some discussion here about this version: https://github.com/dotnet/corefx/pull/30949. /cc @eerhardt.

Reply:
corefx needs this version because of https://github.com/dotnet/core-setup/issues/4291.

I'm not too concerned here in the 'master' branch on staying on N-1.  `2.1.401` SDK will ship LONG before `3.0` will ship.

Reply:
Good point, we're already getting the `2.1.401` SDK ready in the `release/2.1` branch. This seems fine to be practical.

Reply:
Agree, `2.1.401` is good for `3.0`